README.rst

```diff
@@ -15,56 +15,57 @@ The Mapbox Python SDK is a low-level client API, not a Resource API such as the
 Services
 ========
 
-- **Analytics** `examples <./docs/analytics.md>`__, `website <https://www.mapbox.com/api-documentation/#analytics>`__
+- **Analytics V1** `examples <./docs/analytics.md>`__, `website <https://www.mapbox.com/api-documentation/#analytics>`__
```

Comment:
I'm putting the supported versions in the readme for visibility.

Reply:
Sounds good @sgillies